Projects/App/Sources/Apply/ApplyViewController.swift

```diff
@@ -13,25 +13,79 @@ import RxSwift
 import SnapKit
 
 final class ApplyViewController: BaseViewController<ApplyViewModelProtocol> {
-    
-    private var gifticonId: Int = 0
     private let disposeBag = DisposeBag()
-    
+    private let scrollView = UIScrollView()
+    private let scrollContentView = UIView()
+    private let applyGifticonView = ApplyGifticonView()
+    private let applyButton = DDIPCTAButton()
+    private let toastView = ToastView()
+
     private lazy var navigationBar: DDIPNavigationBar = {
         return DDIPNavigationBar(
             leftBarItem: DDIPNavigationBar.BarItem.back,
             title: "응모하기",
             rightButtonsItem: nil)
     }()
-    
-    private let scrollView = UIScrollView()
-    
-    private let scrollContentView = UIView()
-    
-    private let applyGifticonView = ApplyGifticonView()
-    
-    private let applyButton = DDIPCTAButton()
-    
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    private func showSuccessToast(category: DDIPAsset.name) {
+        toastView.configureToastView(with: self.view, style: .apply, image: category)
+        toastView.showToastView(with: self.view)
+    }
+
+    private func showFailToast() {
+        toastView.configureToastView(with: self.view, style: .applyFail, image: .iconRotateLogoCharacterEmpty)
+        toastView.showToastView(with: self.view)
+    }
+
+    private func setApplyCompletButtonState() {
+        applyButton.setBackgroundColor(buttonColor: .secondarySkyblue200)
+        applyButton.setTitle(title: "응모 완료!")
+    }
+
+    private func setFinishButtonState() {
+        applyButton.setBackgroundColor(buttonColor: .secondarySkyblue200)
+        applyButton.setTitle(title: "응모 마감")
+    }
+
+    override func bind() {
+        applyButton.rx.tap.asObservable()
```

Comment:
override라면 super를 호출해주는것 잊지마세요 !
```suggestion
    override func bind() {
        super.bind()
        applyButton.rx.tap.asObservable()
```

Reply:
앗.. override method일거라고 인지못했었네요 감사합니다!